src/js/RacingCar.js

```diff
@@ -0,0 +1,57 @@
+import { CAR_PROGRESS_THRESHOLD, CAR_NAME_RULE, TRY_COUNT_LIMIT } from './constants.js';
+
+export default class RacingCar {
+  cars;
+  tryCount;
+
+  constructor(cars = [], tryCount = 0) {
+    this.cars = cars;
+    this.tryCount = tryCount;
+  }
+
+  #isCarNameLengthValid(cars) {
+    return cars.every((name) => name.length >= CAR_NAME_RULE.MIN_LENGTH && name.length <= CAR_NAME_RULE.MAX_LENGTH);
+  }
```

Comment:
```suggestion
  #isCarNameLengthValid(cars) {
    return cars.every((name) => name.length >= RacingCar.MIN_LENGTH && name.length <= RacingCar.MAX_LENGTH);
  }
```
이렇게 아예 RacingCar 라는 class 내에 상수를 선언해줄 수 있지 않을까요?